dpctl/apis/include/dpctl4pybind11.hpp

```diff
@@ -174,6 +192,8 @@ struct dpctl_capi
     std::shared_ptr<py::object> default_usm_memory;
     std::shared_ptr<py::object> default_usm_ndarray;
     std::shared_ptr<py::object> as_usm_memory;
+    std::shared_ptr<py::object> default_sycl_kernel;
```

Comment:
Why are these not initialized like the other private members in the initializer list?

Reply:
A nitpick. In general, `struct` should be restricted to passive objects that carry data and the `dpctl_capi` should be a `class`. Then the private members should follow a naming convention such as `std::shared_ptr<py::object> default_sycl_program_` with the trailing underscore. (Ref: https://google.github.io/styleguide/cppguide.html#Variable_Names)

Reply:
Thanks @diptorupd. I have applied suggested changes. Incidentally `default_sycl_kernel` was unused, so I removed it. Thanks for spotting it.